pkg/apis/apps/types.go

```diff
@@ -206,7 +206,6 @@ type StatefulSetSpec struct {
 	// Minimum number of seconds for which a newly created pod should be ready
 	// without any of its container crashing for it to be considered available.
 	// Defaults to 0 (pod will be considered available as soon as it is ready)
-	// This is an alpha field and requires enabling StatefulSetMinReadySeconds feature gate.
```

Comment:
@liggitt - Should I open backports to make this beta in previous releases?

Reply:
probably so, since the API docs are incorrect